test/conftest.py

```diff
@@ -40,9 +41,21 @@ def get_test_config() -> dict[str, Any]:
     for nameval in config_parser.items("SetAPI"):
         cfg_dict[nameval[0]] = nameval[1]
 
+    return cfg_dict
+
+
+@pytest.fixture(scope="session")
+def ctx(config: dict[str, str]) -> dict[str, str | list]:
```

Comment:
Finally breaking the KBase mold and renaming this to `context` or `api_context` might make it more clear what it's about.

I feel like you did something like that in another PR? Maybe a later one? I forget.

Still, just an idea, not a requirement.

Reply:
I might have done it in another repo? Anyway, I'll implement your suggestion in a later PR to avoid having to do a load of rebasing. :)